src/openms/include/OpenMS/CHEMISTRY/RNaseDigestion.h

```diff
@@ -52,12 +52,24 @@ namespace OpenMS
   class OPENMS_DLLAPI RNaseDigestion: public EnzymaticDigestion
   {
   public:
+    /// Constructor
+    RNaseDigestion();
+
+    /// Destructor
+    virtual ~RNaseDigestion() = default;
+
     /// Sets the enzyme for the digestion
     void setEnzyme(const DigestionEnzyme* enzyme) override;
 
     /// Sets the enzyme for the digestion (by name)
     void setEnzyme(const String& name);
 
+    /// Sets how many As may be modified to I
```

Comment:
maybe extend documentation here with examples of enzymes / why this has been added

Reply:
Added: "This is relevant for RNase T1 cleavage when variable inosine modifications are considered."